docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Mintlify Starter Kit
```

Comment:
Just want to make sure, this doesn't show up on the deployed site right? I have a task to make a CONTRIBUTING.md, I'll make a note to move this content there.

Reply:
Yep doesn't show up!